### PR TITLE
RR-738 - Check your answers page

### PR DESCRIPTION
**Pull request by mattwills-moj-digital**
This PR gets the short induction journey as far as the Check Your Answers page (which submits ok, but doesn't yet render correctly). I've added TODOs where necessary.

Good luck with getting this over the line @nathanrussell-moj-digital! Sorry I couldn't quite finish it! 😄

### CODE REVIEW

File: server/routes/induction/update/checkYourAnswersUpdateController.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import createError from 'http-errors'
+import CheckYourAnswersController from '../common/checkYourAnswersController'
+import toCreateOrUpdateInductionDto from '../../../data/mappers/createOrUpdateInductionDtoMapper'
+import logger from '../../../../logger'
+import { InductionService } from '../../../services'
+
+/**
+ * Controller for the Check Your Answers screen after updating an Induction.
+ */
+export default class CheckYourAnswersUpdateController extends CheckYourAnswersController {
```
Comment (mattwills-moj-digital) [inline server/routes/induction/update/checkYourAnswersUpdateController.ts:11]: I didn't have time to write a test for this (even though I would normally write one first! 😂 )

Comment (nathanrussell-moj-digital) [reply server/routes/induction/update/checkYourAnswersUpdateController.ts:11]: I've added RR-766 to address this

File: server/routes/induction/update/qualificationsListUpdateController.ts [hunk 2]
```diff
@@ -18,6 +18,11 @@ export default class QualificationsListUpdateController extends QualificationsLi
     const { prisonNumber } = req.params
     const { pageFlowHistory } = req.session
     if (pageFlowHistory) {
+      // TODO - The intention here is to revert the position of the current page index to /qualifications or /want-to-add-qualifications (effectively so that /qualification-level and /qualification-detail are removed, if applicable)
```
Comment (nathanrussell-moj-digital) [inline server/routes/induction/update/qualificationsListUpdateController.ts:21]: I've created RR-767 to address this

File: server/views/pages/induction/checkYourAnswers/partials/_educationAndTraining.njk [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        <div data-qa="additionalTraining-{{ loop.index }}">{{ item | formatAdditionalTraining }}{{ ' - ' + inductionDto.previousTraining.trainingTypeOther if item === 'OTHER' }}{{ ',' if inductionDto.previousTraining.trainingTypes.length !== loop.index }}</div>
+      {% endfor %}
+    </dd>
+    <dd class="govuk-summary-list__actions">
+      <a class="govuk-link govuk-link--no-visited-state" href="/prisoners/{{ prisonerSummary.prisonNumber }}/induction/additional-training" data-qa="additionalTrainingLink">
+        Change<span class="govuk-visually-hidden"> other training or qualifications they have</span>
+      </a>
+    </dd>
+  </div>
+  <div class="govuk-summary-list__row">
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/induction/checkYourAnswers/partials/_educationAndTraining.njk:62]: Weird empty row!
Given that the scope of what we are doing is to take the views and functionality verbatim from the CIAG UI, then to that end that is what we have done here
I'm guessing it was added to add spacing perhaps? But I'm not going to change it as part of these stories.

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**